modules/library/referencing/src/main/java/org/geotools/referencing/factory/wms/Auto97001.java

```diff
@@ -0,0 +1,99 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ * 
+ *    (C) 2004-2008, Open Source Geospatial Foundation (OSGeo)
```

Comment:
The file has been added in 2015, so this line should be:
``(C) 2015, Open Source Geospatial Foundation (OSGeo)``